### ISSUE TITLE
Multi Tablist

### ISSUE THREAD

**Issue by DJClement50** [CodeCrafter47/BungeeTabListPlus#179]
Good evening,
I followed your wiki to create more tablist but nothing happens on my servers

Thank you for your reply

**Comment by CodeCrafter47** [comment]
That's not much information to tell what's wrong. Using multiple tab lists works fine for me.

**Comment by DJClement50** [comment]
Yes it works fine for you may be for me but I do not know how to install a different tablist per servers I followed your wiki but nothing

**Comment by CodeCrafter47** [comment]
Why don't you have a look at the proxy log. BungeeTabListPlus logs which tablist configuration files it loads on startup and on reload. You will find a few lines like these:
```
17:25:18 [INFO] [BungeeTabListPlus] Loading custom-6.yml
17:25:18 [INFO] [BungeeTabListPlus] Loading default.yml
```
If your custom file is listed there then the issue is inside the file itself, if it isn't listed there then you saved it at the wrong place or used the wrong file extension.

**Comment by DJClement50** [comment]
Yes its loaded in the console but on the network (I have 5 servers in dedicated) and I want the pvp do not have the same display as the lobby or survival

**Comment by CodeCrafter47** [comment]
What do you have the showTo and priority options set to for the files?
